archive/level21-Additions.lark

```diff
@@ -1,10 +1,10 @@
 // adds length of lists
 
 print: _PRINT (_SPACE* _LEFT_BRACKET _SPACE* (quoted_text | list_access | var_access | sum | length) (_SPACE (quoted_text | list_access | var_access | sum | length))* _RIGHT_BRACKET)?
-input: var _SPACE _IS _SPACE _INPUT (_LEFT_BRACKET  (quoted_text | list_access | var_access | sum | length) (_SPACE (quoted_text | list_access | var_access | sum | length))*  _RIGHT_BRACKET)?
+input: var _SPACE _IS _SPACE _INPUT _SPACE? (_LEFT_BRACKET  (quoted_text | list_access | var_access | sum | length) (_SPACE (quoted_text | list_access | var_access | sum | length))*  _RIGHT_BRACKET)?
```

Comment:
Mostly a question for my own understanding; why is adding this space here required for making Hindi work?

Reply:
Ow good catch @ThomasKluiters, this file should not have been part of the PR at all!

Reply:
Note thought that this is in the archive folder that is not actively used but an artefact of changes in the past that still might need to be places somehere